(Update) Added new limitation (#7297)
harness/developer-hub: docs/platform/templates/create-a-secret-manager-template.md

* You must have **Create/Edit** permissions for secrets.
* You must have **Create/Edit** permissions for connectors.

> ### Limitations

Reviewer: ```suggestion
### Limitations

```

> * If you are intending to include Secrets Variable references within the template, the secrets must exist within Harness Built-In Secrets manager.  References to secrets held in other managers.  For example, if you refer to a secret within AWS Secrets manager, you will encounter the following message 

Reviewer: ```suggestion
* If you want to include Secrets Variable references within the template, the secrets must exist within the Harness Built-In Secrets manager. References to secrets within other managers might cause errors. For example, if you refer to a secret within the AWS Secrets manager, you will encounter the following message.

```
